src/styles/block/stopwatch.scss

```diff
@@ -0,0 +1,50 @@
+.stopwatch {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+
+  &__border {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    width: 200px;
```

Comment:
**Demo has 80vmin size**

Reply:
but as i understand. i can use the sizes that i want to? :)